_posts/0000-01-02-faith098.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
A duck walked up to a lemonade stand and he said to the man running the stand....hey, got any grapes?